include/fuse.h

```diff
@@ -195,7 +206,7 @@ struct fuse_config {
 	 * Note that this does *not* affect the inode that libfuse 
 	 * and the kernel use internally (also called the "nodeid").
 	 */
-	int use_ino;
+	uint32_t use_ino;
```

Comment:
You are changing signed int to unsigned int. Based on the commit message, which states `int -> int32_t`, this does not look intentional? Other members are also affected.

Reply:
@mxmlnkn Yeah, was intended, but I was not done yet with the change. Initially I wanted change to `uint32_t field : 1;`, but I had forgotten that some values are exposed as parameters in fuse.c - resulted in removing `:1`. I updated these to uint8_t now. Also updated the commit message. 
Thanks for your review! Will review the entire series myself in the morning and then add other reviewers. I hope we get 3.17 out this week.